patches/4D5308BC - Crackdown 2 (TU0).patch.toml

```diff
@@ -0,0 +1,192 @@
+
+    [[patch.be8]] # Enable GPU Hz
+        address = 0x836c6e20
+        value = 0x01
+
+    [[patch.be8]] # Red Text
+        address = 0x836c614e
+        value = 0x01
+
+[[patch]]
+    name = "Show CPU/GPU Performance Graphs"
+    desc = "This will show FPS and Performance Graphs."
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]] # Enable CPU/GPU Performance Graphs
+        address = 0x836c6e21
+        value = 0x01
+
+    [[patch.be8]] # Enable Display Render Info
+        address = 0x836c6f2c
+        value = 0x01
+
+    [[patch.be8]] # Enable GPU Hz
+        address = 0x836c6e20
+        value = 0x01
+
+    [[patch.be8]] # Red Text
+        address = 0x836c614e
+        value = 0x01
+
+[[patch]]
+    name = "Light Bloom"
+    desc = "Toggle fullbright, may improve performance."
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82ffd0bf
+        value = 0x01
+
+[[patch]]
+    name = "Toggle Shadows"
+    desc = "May improve performance."
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82ffd0c4
+        value = 0x00
+
+[[patch]]
+    name = "Helicam / Fly around the map"
+    desc = "Teleport your character. Cheat Engine address: 1836C615B"
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x836c615b
+        value = 0x01
+
+[[patch]]
+    name = "Toggle Map Outlines"
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82ffd0b5
+        value = 0x01
+
+[[patch]]
+    name = "Outline Mode"
+    desc = "Must have outlines enabled."
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be32]] # White Outline
+        address = 0x836c66f8
+        value = 0x00000002
+
+[[patch]]
+    name = "Opaque Pass / Dark Mode"
+    desc = "Disable Rendering"
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82ffca8b
+        value = 0x00
+
+[[patch]]
+    name = "Dark Mode"
+    desc = "Map stencil with white outlines in dark mode."
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]   # Opaque Pass / Dark Mode
+        address = 0x82ffca8b
+        value = 0x00
+
+    [[patch.be8]]   # Enable Outlines
+        address = 0x82ffd0b5
+        value = 0x01
+
+    [[patch.be32]]  # Enable Outline Mode - White Outline
+        address = 0x836c66f8
+        value = 0x00000002
+
+    [[patch.be8]]   # Disable Alpha Render Pass
+        address = 0x82ffd0b4
+        value = 0x00
+
+    [[patch.be8]]   # Enable Particle Alpha
+        address = 0x836c68b5
+        value = 0x01
+
+    [[patch.be8]]   # Disable Light Bloom for stronger white outline
+        address = 0x82ffd0bf
+        value = 0x00
+
+[[patch]]
+    name = "Demo Mode"
+    desc = "Alternatively create a demo.txt next to default.xex"
+    author = "Adrian"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x836c6daf
+        value = 0x01
```

Comment:
```suggestion
    [[patch.be32]] # Bottomless Clip
        address = 0x822056bc
        value = 0x60000000

    [[patch.be8]] # Ammo Display
        address = 0x82244810
        value = 0x41

[[patch]]
    name = "God Mode"
    author = "Adrian, Crippy"
    is_enabled = false

    [[patch.be32]]
        address = 0x82223e9c
        value = 0x60000000
    [[patch.be32]]
        address = 0x825354c0
        value = 0x60000000

[[patch]]
    name = "On-screen FPS counter"
    author = "Adrian"
    is_enabled = false

    [[patch.be8]] # Enable Display Render Info
        address = 0x836c6f2c
        value = 0x01
    [[patch.be8]] # Disable Performance Graphs
        address = 0x836c6e21
        value = 0x00
    [[patch.be8]] # Enable GPU Hz
        address = 0x836c6e20
        value = 0x01
    [[patch.be8]] # Red Text
        address = 0x836c614e
        value = 0x01

[[patch]]
    name = "Show CPU/GPU Performance Graphs"
    desc = "This will show FPS and Performance Graphs."
    author = "Adrian"
    is_enabled = false

    [[patch.be8]] # Enable CPU/GPU Performance Graphs
        address = 0x836c6e21
        value = 0x01
    [[patch.be8]] # Enable Display Render Info
        address = 0x836c6f2c
        value = 0x01
    [[patch.be8]] # Enable GPU Hz
        address = 0x836c6e20
        value = 0x01
    [[patch.be8]] # Red Text
        address = 0x836c614e
        value = 0x01

[[patch]]
    name = "Light Bloom"
    desc = "Toggle fullbright, may improve performance."
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]
        address = 0x82ffd0bf
        value = 0x01

[[patch]]
    name = "Toggle Shadows"
    desc = "May improve performance."
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]
        address = 0x82ffd0c4
        value = 0x00

[[patch]]
    name = "Helicam / Fly around the map"
    desc = "Teleport your character. Cheat Engine address: 1836C615B"
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]
        address = 0x836c615b
        value = 0x01

[[patch]]
    name = "Toggle Map Outlines"
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]
        address = 0x82ffd0b5
        value = 0x01

[[patch]]
    name = "Outline Mode"
    desc = "Must have outlines enabled."
    author = "Adrian"
    is_enabled = false

    [[patch.be32]] # White Outline
        address = 0x836c66f8
        value = 0x00000002

[[patch]]
    name = "Opaque Pass / Dark Mode"
    desc = "Disable Rendering"
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]
        address = 0x82ffca8b
        value = 0x00

[[patch]]
    name = "Dark Mode"
    desc = "Map stencil with white outlines in dark mode."
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]   # Opaque Pass / Dark Mode
        address = 0x82ffca8b
        value = 0x00
    [[patch.be8]]   # Enable Outlines
        address = 0x82ffd0b5
        value = 0x01
    [[patch.be32]]  # Enable Outline Mode - White Outline
        address = 0x836c66f8
        value = 0x00000002
    [[patch.be8]]   # Disable Alpha Render Pass
        address = 0x82ffd0b4
        value = 0x00
    [[patch.be8]]   # Enable Particle Alpha
        address = 0x836c68b5
        value = 0x01
    [[patch.be8]]   # Disable Light Bloom for stronger white outline
        address = 0x82ffd0bf
        value = 0x00

[[patch]]
    name = "Demo Mode"
    desc = "Alternatively create a demo.txt next to default.xex"
    author = "Adrian"
    is_enabled = false

    [[patch.be8]]
        address = 0x836c6daf
        value = 0x01
```